### ISSUE TITLE
Content is clipped or overlapped when text spacing is adjusted

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#58348]
### Issue Description:

The Rating stars are clipped when text spacing is adjusted.
 

The site design must be flexible to accommodate user-defined text resizing, ensuring readability for low-vision users or those with reading disabilities who may have set larger text sizes or increased spacing in their browsers.
 
 
<img width="346" alt="Image" src="https://github.com/user-attachments/assets/406dd07c-5739-4194-9521-9ca6e2b2a5a9" />

### Component or Page: 

Component: Filters (v.9.9)    

### Name affected element:

Rating stars

### Relevant code snippets:

```
<div class=""wc-block-components-product-rating__stars"" role=""img"" aria-label=""Rated 5 out of 5"">
				<span style=""width: 100%"" aria-hidden=""true""></span>
			</div>
```

### Recommended fix:

Ensure users can adjust text spacing to make it easier to read: line-height, letter spacing, word spacing, paragraph spacing.
 There should be no loss of content or functionality occurring by setting all of the following and by changing no other style property:
 -Line height (line spacing) to at least 1.5 times the font size;
 -Spacing following paragraphs to at least 2 times the font size;
 -Letter spacing (tracking) to at least 0.12 times the font size;
 -Word spacing to at least 0.16 times the font size.
 
 Ensure the content adapts to adjusted text spacing. Ensure that the height and width of content containers are specified using dynamic units like  "em".
 
 For more information refer to https://www.w3.org/WAI/WCAG22/Understanding/text-spacing.html

**Comment by dinhtungdu** [comment]
@vikrampm1 How did you change the spacing of the rating? Did you use custom CSS or something? I'm asking as, currently, we don't support changing style setting of rating star of the Rating filter and I want to understand your setup.

**Comment by vikrampm1** [comment]
Hi @dinhtungdu these issues were reported by the EqualizeDigital team so I'm looping them here to see how the spacing was changed. cc @amberhinds @majomaldonado

**Comment by MajoMaldonado** [comment]
Hi @dinhtungdu we use [this bookmarklet](https://codepen.io/stevef/full/YLMqbo) to test the text spacing.
